composer.json

```diff
@@ -73,7 +73,8 @@
             "*": "dist"
         },
         "allow-plugins": {
-            "php-http/discovery": true
+            "php-http/discovery": true,
+            "cweagans/composer-patches": true
```

Comment:
Why this package was added?

Reply:
Oh, by mistake it got added.
Thanks @mxr576